cadc-util/src/main/java/ca/nrc/cadc/auth/IdentityType.java

```diff
@@ -112,5 +129,18 @@ public String toString()
         return this.getClass().getSimpleName() + "[" + value + "]";
     }
 
+
+    public static IdentityType get(String identity) {
```

Comment:
Isn't this get(String) duplicating toValue(String)? In general enums are small enough to not need a map to implement so the toValue impl is fine.